bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/models/v1/ClientLibrariesImpl.java

```diff
@@ -167,6 +166,13 @@ public String getJsAndCssIncludes() {
         return getLibIncludes(null);
     }
 
+    /**
+     * Returns the markup for including the client libraries into an HTML page
+     *
+     * @param type - the type of the client libraries
+     *
+     * @return Markup for including the client libraries
```

Comment:
I'd phrase it as: Markup that includes the client libraries

Reply:
Done in 000c479